### PR TITLE
Camera Modified.... Now After Take Image app not crash

### PR DESCRIPTION
**Pull request by SamarpanCoder2002**
Now app will not crash when take image from camera....(Issue related: #210 )

Video Demo: https://drive.google.com/file/d/1RDKyyhrpEoUwKSvF-aPOlmST7cLVzfB6/view?usp=sharing

I am also a GSSOC'21 participant.....

Thank you @anushbhatia

### CODE REVIEW

**Comment by SamarpanCoder2002** [comment]
hello @anushbhatia .... if it looks good..kindly merge it .... demo video also provided...

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
@SamarpanCoder2002 can you please go through all the failed lint checks once and fix them?

**Comment by SamarpanCoder2002** [comment]
I think that problem lint check problems related to platform specific OS.... Problems arrive in ubuntu, windows, macos.....But that issues not problem the application.... But some of the isuues, I can't modified, as in one issue, it requested to use is not empty instead of length.... But at there, I have to write length to setup the logic..... So I think that will be not a problem to merge it.... because it works fine..
Thanks @anushbhatia

**Comment by Saransh-cpp** [comment]
> I think that problem lint check problems related to platform specific OS.... Problems arrive in ubuntu, windows, macos.....But that issues not problem the application.... But some of the isuues, I can't modified, as in one issue, it requested to use is not empty instead of length.... But at there, I have to write length to setup the logic..... So I think that will be not a problem to merge it.... because it works fine..
> Thanks @anushbhatia

The checks are failing because of some minor mistakes in the code (not necessarily errors but maybe a thing that can be replaced with something more efficient or a keyword that won't make a big difference right now but should be added), you can start by solving the smaller issues first and then figure out how to deal with the other issues. Some small issues - 
1. File name should be in small letters (something_like_this.dart).
2. Placing the `'dart'` imports before any other imports.
3. Using `const` wherever a constant constructor is defined or called.

File: lib/cameraOperation.dart [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+              ),
+            ),
+            Expanded(
+              child: FloatingActionButton(
+                heroTag: '2',
+                backgroundColor: Colors.white10,
+                onPressed: () {
+                  _onCapturePressed(context);
+                },
+                child: Icon(Icons.camera_alt),
```
Comment (Saransh-cpp) [inline lib/cameraOperation.dart]: Add `const` before `Icon()`

File: lib/cameraOperation.dart [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        padding: const EdgeInsets.all(8.0),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Expanded(
+              child: FloatingActionButton(
+                heroTag: '1',
+                backgroundColor: Color.fromRGBO(0, 0, 0, 0.0),
+                onPressed: _onSwitchCamera,
+                child: Icon(Icons.flip_camera_android_rounded),
```
Comment (Saransh-cpp) [inline lib/cameraOperation.dart]: Add `const` before `Icon()`

File: lib/cameraOperation.dart [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                  Navigator.pop(context);
+                  final PickedFile imageFile =
+                      await ImagePicker().getImage(source: ImageSource.gallery);
+                  if (imageFile == null) return;
+                  final File tmpFile = File(imageFile.path);
+
+                  Navigator.of(context).push(MaterialPageRoute(
+                      builder: (context) => Imageview(tmpFile, images)));
+                },
+                child: Icon(
```
Comment (Saransh-cpp) [inline lib/cameraOperation.dart]: Add `const` before `Icon()`

File: lib/cameraOperation.dart [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+      floatingActionButton: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Expanded(
+              child: FloatingActionButton(
+                heroTag: '1',
+                backgroundColor: Color.fromRGBO(0, 0, 0, 0.0),
```
Comment (Saransh-cpp) [inline lib/cameraOperation.dart]: Add `const` before `Color`

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
These are some of the small changes that will get you started with passing the checks

**Comment by SamarpanCoder2002** [comment]
@anushbhatia Now all checks are passes and it looks fine....You can check again the demo video link.....I am also a GSSOC'21 participant ...... Please merge it ....Thank u

and @Saransh-cpp ...thanks a lot for help me

**Comment by anushbhatia** [comment]
Screenshot_20210326_013616_com example doclense
My camera resolution is afted by this

**Pull request closed without merging**